tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -1,7 +1,7 @@
 @api @skipOnOcV10
 Feature: Change data of space
-  As a user
-  I want to be able to change the data of the created space(increase the quota, change name, etc.)
+  As an admin user
```

Comment:
As a user with "admin rights". There will be cases with other roles later

Reply:
fixed